tracee-ebpf/tracee/tracee.go

```diff
@@ -114,7 +115,7 @@ type CaptureConfig struct {
 	Exec            bool
 	Mem             bool
 	Profile         bool
-	NetIfaces       []string
```

Comment:
are you sure about this? this is user facing configuration, in similar cases (for example, output path) we always ask for a simple type (string) and do the necessary work internally

Reply:
I changed it back to slice of string. Thanks